src/support/azext_support/aaz/latest/support/in_subscription/communication/_create.py

```diff
@@ -58,7 +58,6 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema.body = AAZStrArg(
             options=["--body"],
             help="Body of the communication.",
```

Comment:
Remove edits made to this file and all other communication and chat transcript files